mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -498,7 +498,8 @@ struct ElementwiseOpFusionResult {
   Operation *fusedOp;
   llvm::DenseMap<Value, Value> replacements;
   static llvm::SmallDenseSet<int>
-  getPreservedProducerResults(GenericOp producer, GenericOp consumer);
+  getPreservedProducerResults(GenericOp producer, GenericOp consumer,
```

Comment:
Actually looking at this now, I dont see a reason for this to be a method in this struct. While here, could you just move it to a separate method outside of this struct.

Reply:
Fixed
A comment was added in order to note that there is a chance of disagreement between the implementation of the transformation and the result of the method. And I moved that method outside of the struct.